syntaxes/odoc.json

```
@@ -0,0 +1,81 @@
+{
+  "scopeName": "source.ocaml.odoc",
+  "name": "comment.other.odoc",
+  "fileTypes": ["mld"],
+  "patterns": [
+    {
+      "name": "comment.other.odoc",
+      "begin": "",
+      "end": "",
+      "patterns": [{ "include": "#markup" }]
+    },
```

I think everything except this and `odoc documentation tag` should be in `markup`.

---

Actually I think this rule is actually unnecessary since the comment colors comes from the OCaml syntax.

You should be to replace this with just

```suggestion
    { "include": "#markup" },
```